multimedia/VLC/files/vlc

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
no you're right. This was used before I adopted Mojca's approach to create the file on the fly